dotnet-test-samples/kinesis-lambda-dynamodb/tests/KinesisEventHandler.IntegrationTests/ProcessEmployeeTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Net;
+using FluentAssertions;
+using KinesisEventHandler.IntegrationTests.Utilities;
+using xRetry;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace KinesisEventHandler.IntegrationTests;
+
+[TestCaseOrderer(
+    ordererTypeName: "KinesisEventHandler.IntegrationTests.Utilities.PriorityOrderer",
+    ordererAssemblyName: "KinesisEventHandler.IntegrationTests")]
+public class ProcessEmployeeTests : IClassFixture<ProcessEmployeeFixture>, IDisposable
+{
+    private readonly ProcessEmployeeFixture _processEmployeeFixture;
+    private readonly ITestOutputHelper _testOutputHelper;
+    private const string EmployeeId = "569c13fc-1435-45cb-847d-38e89a86e5a0";
+    private bool _disposed;
+
+    public ProcessEmployeeTests(ProcessEmployeeFixture processEmployeeFixture, ITestOutputHelper testOutputHelper)
+    {
+        _processEmployeeFixture = processEmployeeFixture;
+        _testOutputHelper = testOutputHelper;
+    }
+
+    public void Dispose()
+    {
+        if (_disposed)
+        {
+            return;
+        }
+
+        _disposed = true;
+        _processEmployeeFixture.KinesisClient.Dispose();
+    }
+
+    [Fact, TestPriority(1)]
+    public async Task WriteToEmployeeRecordsStream_Should_ReturnSuccess()
```

Comment:
Is there a way to avoid using test ordering, the more I see it the more I'm sure that in a real world scenarios our customers would get test failures due to tests shared states, it starts to emerge as a pattern for all of our integration tests and I'm afraid we are teaching bad practices

Reply:
Removed ordering, only alternative with integration testing distributed applications is long polling. Implemented.